README.md

```diff
@@ -95,3 +95,52 @@ Duplicate word: is
 
 Evening Session
 
+Exercise 1
+
+Modify your program to create a list of users' names, ages, emails,
+phone numbers, and addresses. Save it to a CSV file using FileWriter.
+Each line should represent one user's data.
+
+Exercise 2
```

Comment:
love the readme file. makes things much easier to find. continue please with this practice